tests/classes/test_hypergraph.py

```diff
@@ -80,6 +80,12 @@ def test_neighbors(edgelist1, edgelist2):
     assert H2.neighbors(4) == {3, 5, 6}
     assert H2.neighbors(1) == {2}
 
+    # test edge neighbors
+    assert H1.neighbors(0, edges=True) == set()
+    assert H1.neighbors(1, edges=True) == set()
+    assert H1.neighbors(2, edges=True) == {3}
+    assert H1.neighbors(3, edges=True) == {2}
```

Comment:
What do you think about `H.nodes.neighbors` and `H.edges.neighbors` instead?

Reply:
I like this idea! I guess that this should get implemented on the `feat_nodestats` branch?

Reply:
Yep!